package.json

```diff
@@ -54,7 +54,7 @@
     "eslint": "^7.11.0",
     "eslint-config-airbnb": "^18.2.0",
     "eslint-config-prettier": "^6.13.0",
-    "eslint-plugin-backpack": "^2.0.1",
+    "eslint-plugin-backpack": "^2.0.2",
```

Comment:
Do we need to do this? It's already in range, right?

Reply:
Its good to set that given that we want the minimum to support the correct updated tokens 😄

Reply:
Plus if I don't do it now Dependabot will anyway :P